content/04.study.md

```diff
@@ -47,6 +47,24 @@ Deep learning applied to gene expression data is still in its infancy, but the f
 Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
 For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.
 
+### DNA Methylation
+
+#### Inference, Imputation, and Prediction
+
+Deep learning approaches are beginning to help address some of the current limitations of feature-by-feature analysis approaches to DNA methylation data, and may help uncover additional important features necessary to understand the biological underpinnings behind different pathological states.
+One of the more popular applications is the prediction of the degree of methylation at CpG sites neighboring measured sites.
```

Comment:
```suggestion
One of the more popular applications is imputing the degree of methylation at CpG sites that are within a few thousand base pairs of measured sites.
```
Is this appropriate? I think it would be helpful to include some idea of what range the methods are trying to predict. Feel free to re-word.

Reply:
I think this may be appropriate enough. For the AE methods, it would be appropriate to add that either within a few thousand sites (one sample) or informed by similar samples.